templates/customers/reset_password.liquid

```diff
@@ -60,12 +60,13 @@
           <svg aria-hidden="true" focusable="false" role="presentation">
             <use href="#icon-error" />
           </svg>
-          {{ form.errors.translated_fields['password'] | capitalize }} {{ form.errors.messages['password'] }}
+          {{ form.errors.translated_fields['password'] | capitalize }}
+          {{ form.errors.messages['password'] }}
         </small>
```

Comment:
Same here but I don't if it's something the plugin could catch and tweak 🤔  It's probably more on us to know and make sure that inline elements will add white space in those cases.
![](https://screenshot.click/19-13-exw9h-9418u.png)